turbopuffer/__init__.py

```diff
@@ -2,6 +2,7 @@
 import sys
 api_key = os.environ.get('TURBOPUFFER_API_KEY')
 api_base_url = os.environ.get('TURBOPUFFER_API_BASE_URL', 'https://api.turbopuffer.com')
+upsert_vectors_as_base64 = os.environ.get('TURBOPUFFER_UPSERT_VECTORS_AS_BASE64', 'true').lower() == 'true'
```

Comment:
Ah, I didn't realize we had some other more esoteric config values in here as globals. Good call to keep this one consistent, @pushrax! 👍🏽